src/Config/YamlConfigFileLoader.php

```diff
@@ -123,9 +138,7 @@ private function extractData(string $key, array $rawConfig, $default = false)
      */
     private function loadFileContents(string $file): string
     {
-        $contents = file_get_contents($file);
-
-        return $contents;
+        return file_get_contents($file);
```

Comment:
Any reason to keep that private method that is just a proxy to file_get_contents with another name?